lib/mini_profiler/config.rb

```diff
@@ -13,7 +13,7 @@ def self.attributes
     end
 
     attr_accessor :authorization_mode, :auto_inject, :backtrace_ignores, :backtrace_includes, :backtrace_remove,
```

Comment:
Should I also reformat this list of attrs to be a certain max width per line, too? (80 cols or 100 cols?) Or leave it?